### PR TITLE
Industrial grinder

### PR DESCRIPTION
**Pull request by PlinHost**
## About The Pull Request
Added industrial grinder that processes items like regular grinder, but instead of a detachable beaker it has internal  buffer, from which bottles can be created like in chemmaster. Bottles are also created automatically when there is enough of a single reagent to fill one.

Fixed grinder: it is now constructed using machine frame, not computer one, and is deconstructible.

## Screenshots
<details>
  <summary>Expand</summary>

img
</details>

### CODE REVIEW

**Comment by PlinHost** [comment]
PR updated:
- Refactored grinders to have common parent. Updated paths on map.
- Added industrial grinder circuit board, found in random spawns.
- Fixed grinder construction. It now requires a machine frame, not a computer one.
- Added deconstruction for both grinders.

File: code/modules/reagents/machinery/grinder.dm [hunk 1]
```diff
@@ -1,130 +1,48 @@
...
-	circuit = /obj/item/weapon/circuitboard/reagentgrinder
-	var/inuse = 0
-	var/obj/item/weapon/reagent_containers/beaker = null
+	var/nano_template
 	var/limit = 10
 	var/list/holdingitems = list()
 	var/list/sheet_reagents = list(
 		/obj/item/stack/material/iron = "iron",
-		/obj/item/stack/material/uranium = MATERIAL_URANIUM,
+		/obj/item/stack/material/uranium = "uranium",
```
Comment (TorinoFermic) [inline code/modules/reagents/machinery/grinder.dm]: What happened to this ?

Comment (PlinHost) [reply code/modules/reagents/machinery/grinder.dm]: These defines are for materials, not reagents. It just happens that some reagents have the same name as a material.
This line refers to https://github.com/discordia-space/CEV-Eris/blob/ed2544a340855e1de7de51a1516ed0c4a55e419b/code/modules/reagents/reagents/other.dm#L169-L171 and not to https://github.com/discordia-space/CEV-Eris/blob/ed2544a340855e1de7de51a1516ed0c4a55e419b/code/modules/materials/materials.dm#L244-L245
Whoever added material defines must have placed them here by mistake.

**Comment by SpadesSlick** [comment]
Sprite is coming soon.

**Comment by PlinHost** [comment]
Added icon.

**Pull request merged**